politeiawww/cmd/politeiawwwcli/commands/setdccstatus.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package commands
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"strings"
+
+	cms "github.com/decred/politeia/politeiawww/api/cms/v1"
+)
+
+type SetDCCStatusCmd struct {
+	Args struct {
+		Token  string `positional-arg-name:"token"`
+		Status string `positional-arg-name:"status"`
+		Reason string `positional-arg-name:"reason"`
+	} `positional-args:"true" optional:"true"`
+}
+
+func (cmd *SetDCCStatusCmd) Execute(args []string) error {
+	DCCStatus := map[string]cms.DCCStatusT{
+		"rejected": cms.DCCStatusRejected,
+		"approved": cms.DCCStatusApproved,
+	}
+	// Check for user identity
+	if cfg.Identity == nil {
+		return errUserIdentityNotFound
+	}
+
+	status, ok := DCCStatus[strings.ToLower(cmd.Args.Status)]
+	if !ok {
+		return fmt.Errorf("Invalid status: %v", cmd.Args.Status)
```

Comment:
Can we change this to:

```
		return fmt.Errorf("Invalid status: '%v'.  "+
			"Valid statuses are:\n"+
			"  rejected  reject the DCC\n"+
			"  approved  approve the DCC",
			cmd.Args.Status)
```

So that the user knows what choices are.